packages/react-native-auth/src/hooks/useQRCodeString/useQRCodeString.ts

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { toString as toQRCodeString, QRCodeToStringOptions } from 'qrcode';
+
+import { isFunction } from '@aws-amplify/ui';
+
+export type UseQRCodeStringParams = {
+  onError?: (err: string) => void;
```

Comment:
Do we have a general pattern or philosophy on whether it should be `onError?: (error: Error) => void;` or `onError?: (err: string) => void;`?

Reply:
Good question 😄 We do not, for internal hooks personally have been leaning towards `(errorMessage: string) => void` for ease of use in client code but don't feel particularly strongly either way